api/src/main/java/cz/muni/fi/pa165/facade/ManufacturerFacade.java

```diff
@@ -0,0 +1,27 @@
+package cz.muni.fi.pa165.facade;
+
+import cz.muni.fi.pa165.dto.ManufacturerDTO;
+
+import java.util.Collection;
+import java.util.Date;
+
+/**
+ * @author Martin Sumera
+ */
+public interface ManufacturerFacade {
+
+    void createManufacturer(ManufacturerDTO manufacturerDTO);
+
+    void markAllProducedBottlesSinceDateAsToxic(ManufacturerDTO manufacturer, Date date);
+
+    boolean hasToxicProduction(Long id);
+
+    boolean hasToxicProduction(ManufacturerDTO manufacturer);
+
+    Collection<ManufacturerDTO> findAll();
```

Comment:
Mozno by bolo lepsie davat sem rovno List<ManufacturerDTO> kedze DAO vracia List, to iste pre service